tracking/models.py

```diff
@@ -7,6 +7,7 @@
 
 from django.conf import settings
 
+logger = logging.getLogger(__name__)
```

Comment:
What error were you handling with this addition?

Reply:
The error of `logger` not being define. It's used on the [last line of this file](https://github.com/tl-its-umich-edu/student_explorer/blob/7b21ee2a5d2a8e0403a5e33021f0b47f259e7324/tracking/models.py#L60) and has been in there since 2016 605135fa. I noticed this as an error and not sure if it's always been an error or just recently was flagged more strictly by Python 3.7? 

I think it came up while I was running the test suite. I could have removed the logger command too.